frontend/src/services/http.ts

```diff
@@ -21,8 +23,13 @@ const doRequest = async <T>(url: string, method: string, body?: object | string)
     body
   })
 
+  const HTTP_SERVER_ERROR = 500
```

Comment:
Can you put this variable outside of this function? This way, you can use this constant in any other function. Also, bring the 409 error, put here, and import there in the view component